buildSrc/src/main/kotlin/com/datadog/gradle/plugin/apisurface/ApiSurfacePlugin.kt

```diff
@@ -23,12 +22,6 @@ class ApiSurfacePlugin : Plugin<Project> {
             .create(TASK_GEN_API_SURFACE, GenerateApiSurfaceTask::class.java)
         generateTask.srcDir = File(srcDir, "main")
         generateTask.surfaceFile = surfaceFile
-        target.afterEvaluate {
```

Comment:
it was added...but in the other side: PokoGeneratorConfig ( I moved it there because it was easier to remember what to add as we will have more tasks now to generate code)